packages/frint-react-server/src/renderToString.spec.js

```diff
@@ -7,6 +7,7 @@ import { createApp } from 'frint';
 import { observe, streamProps } from 'frint-react';
 
 import renderToString from './renderToString';
+import Region from '../../frint-react/src/components/Region';
```

Comment:
if you do `npm run bootstrap` from root, you can import modules directly from `frint-react` instead of relative path within the monorepo.